make_ghpages/make_pages.py

```diff
@@ -120,142 +125,161 @@ def get_setup_info(json_url):
     return json_data
 
 
-if __name__ == "__main__":  # noqa: MC0001
-    outdir_abs = os.path.join(pwd, out_folder)
-    static_abs = os.path.join(pwd, static_folder)
+def get_aiida_version(setup_json):
+    """Get AiiDA version that this plugin is compatible with.
+    """
+    import requirements
+
+    if setup_json is None:
+        return None
+
+    try:
+        install_requires = setup_json["install_requires"]
+        reqs = requirements.parse("\n".join(install_requires))
+
+        aiida_specs = []
+        for r in reqs:
```

Comment:
In which order are these read? I see a lot of things like `aiida-core<2.0.0,>=1.0.0b1`, wouldn't it be better to have `>=` first and `<` later? Does this depend on the way people write it in the requirements.txt file?

Reply:
thanks for the notice - I've hardcoded it now